packages/jest-reporters/src/utils.ts

```diff
@@ -133,13 +176,29 @@ export const getSummary = (
       : suitesTotal) +
     ` total`;
 
+  const updatedTestsFailed = `${
+    testsFailed + valuesForCurrentTestCases.numFailingTests
+  } failed`;
+  const updatedTestsPending = `${
+    testsPending + valuesForCurrentTestCases.numPendingTests
+  } skipped`;
+  const updatedTestsTodo = `${
+    testsTodo + valuesForCurrentTestCases.numTodoTests
+  } todo`;
+  const updatedTestsPassed = `${
+    testsPassed + valuesForCurrentTestCases.numPassingTests
+  } passed`;
+  const updatedTestsTotal = `${
+    testsTotal + valuesForCurrentTestCases.numTotalTests
+  } total`;
+
   const tests =
     chalk.bold('Tests:       ') +
-    (testsFailed ? chalk.bold.red(`${testsFailed} failed`) + ', ' : '') +
-    (testsPending ? chalk.bold.yellow(`${testsPending} skipped`) + ', ' : '') +
-    (testsTodo ? chalk.bold.magenta(`${testsTodo} todo`) + ', ' : '') +
-    (testsPassed ? chalk.bold.green(`${testsPassed} passed`) + ', ' : '') +
-    `${testsTotal} total`;
+    (testsFailed ? chalk.bold.red(updatedTestsFailed) + ', ' : '') +
+    (testsPending ? chalk.bold.yellow(updatedTestsPending) + ', ' : '') +
+    (testsTodo ? chalk.bold.magenta(updatedTestsTodo) + ', ' : '') +
+    (testsPassed ? chalk.bold.green(updatedTestsPassed) + ', ' : '') +
+    updatedTestsTotal;
```

Comment:
![image](https://user-images.githubusercontent.com/1404810/88631985-4f841100-d0b3-11ea-976a-d7a05a10a607.png)

So

```ts
const updatedTestsFailed =
  testsFailed + valuesForCurrentTestCases.numFailingTests;
const updatedTestsPending =
  testsPending + valuesForCurrentTestCases.numPendingTests;
const updatedTestsTodo = testsTodo + valuesForCurrentTestCases.numTodoTests;
const updatedTestsPassed =
  testsPassed + valuesForCurrentTestCases.numPassingTests;
const updatedTestsTotal =
  testsTotal + valuesForCurrentTestCases.numTotalTests;

const tests =
  chalk.bold('Tests:       ') +
  (updatedTestsFailed
    ? chalk.bold.red(`${updatedTestsFailed} failed`) + ', '
    : '') +
  (updatedTestsPending
    ? chalk.bold.yellow(`${updatedTestsPending} skipped`) + ', '
    : '') +
  (updatedTestsTodo
    ? chalk.bold.magenta(`${updatedTestsTodo} todo`) + ', '
    : '') +
  (updatedTestsPassed
    ? chalk.bold.green(`${updatedTestsPassed} passed`) + ', '
    : '') +
  `${updatedTestsTotal} total`;
```

You didn't change the condition in the ternary here, but I believe that is necessary?

Reply:
```typescript
const updatedTestsFailed = `${
    testsFailed + valuesForCurrentTestCases.numFailingTests
  } failed`;
  const updatedTestsPending = `${
    testsPending + valuesForCurrentTestCases.numPendingTests
  } skipped`;
  const updatedTestsTodo = `${
    testsTodo + valuesForCurrentTestCases.numTodoTests
  } todo`;
  const updatedTestsPassed = `${
    testsPassed + valuesForCurrentTestCases.numPassingTests
  } passed`;
  const updatedTestsTotal = `${
    testsTotal + valuesForCurrentTestCases.numTotalTests
  } total`;

  const tests =
    chalk.bold('Tests:       ') +
    (testsFailed ? chalk.bold.red(updatedTestsFailed) + ', ' : '') +
    (testsPending ? chalk.bold.yellow(updatedTestsPending) + ', ' : '') +
    (testsTodo ? chalk.bold.magenta(updatedTestsTodo) + ', ' : '') +
    (testsPassed ? chalk.bold.green(updatedTestsPassed) + ', ' : '') +
    updatedTestsTotal;
```

Here , `updatedTestsFailed` is already replaced with `testsFailed` as a condition in ternary
I think I maybe missing something. Could you come again?

Reply:
e.g.

```ts
(testsFailed ? chalk.bold.red(updatedTestsFailed) + ', ' : '')
```

It should reflect the total in progress, so
```ts
(updatedTestsFailed ? chalk.bold.red(updatedTestsFailed) + ', ' : '')
```
But since `updatedTestsFailed` is a string that doesn't work.

So my suggestion is to do
```ts
const updatedTestsFailed = testsFailed + valuesForCurrentTestCases.numFailingTests;
```
on its own, then
```ts
(updatedTestsFailed ? chalk.bold.red(`${updatedTestsFailed} failed`) + ', ' : '')
```

Maybe even


```ts
(updatedTestsFailed > 0 ? chalk.bold.red(`${updatedTestsFailed} failed`) + ', ' : '')
```

to be explicit

Reply:
Ah okayy makes sense now.

Previously I was thinking of displaying `updatedTestsFailed` when we receive `testsFailed` . And update it when a new value of `testsFailed` is passed. But forgot about `valuesForCurrentTestCases.numFailingTests` being passed as well now